app/src/main/java/net/squanchy/service/firebase/FirebaseSquanchyRepository.java

```diff
@@ -71,12 +72,32 @@ public FirebaseSquanchyRepository(DatabaseReference database) {
         return observeChild("sessions", FirebaseEvent.Holder.class);
     }
 
+    public Observable<FirebaseEvent> event(int dayId, int eventId) {
```

Comment:
Will eventually check using a string path thing to get to the interesting child directly as we discussed this morning.